website/content/docs/api-clients/api.mdx

```diff
@@ -79,3 +79,169 @@ The following method conventions are used within Boundary's API:
 ### DELETE
 
 `DELETE` is used for deleting a specific resource, and is only used against a particular resource path.
+
```

Comment:
In this file there is a `Status codes` section. We should update that to include `429` and `503`.

I think we should also include a section on response headers to describe:
- RateLimit
- RateLimitPolicy
- Retry-After